index.html

```diff
@@ -0,0 +1,79 @@
+        type="text/css" />
+    <link href="https://fonts.googleapis.com/css?family=Muli:400,400i,800,800i" rel="stylesheet" type="text/css" />
+    <!-- Core theme CSS (includes Bootstrap)-->
+    <link href="css/styles.css" rel="stylesheet" />
+
+</head>
+
+<body id="page-top">
+    <!-- Navigation-->
+    <nav class="navbar navbar-expand-lg navbar-dark bg-dark fixed-top" id="sideNav">
+        <a class="navbar-brand js-scroll-trigger" href="#page-top">
+            <span class="d-block d-lg-none"> </span>
+            <span class="d-none d-lg-block"><img class="img-fluid img-profile rounded-circle mx-auto mb-2"
+                    src="assets/img/image.png" alt="..." /></span>
+        </a>
+        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarResponsive"
+            aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation"><span
+                class="navbar-toggler-icon"></span></button>
+        <div class="collapse navbar-collapse" id="navbarResponsive">
+            <ul class="navbar-nav">
+                <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#about">About</a></li>
+                <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#experience">Experience</a></li>
+            </ul>
+        </div>
+    </nav>
+    <!-- Page Content-->
+    <div class="container-fluid p-0">
+        <!-- About-->
+        <section class="resume-section" id="about">
+            <div class="resume-section-content">
+                <h1 class="mb-0">
+                  Guincho
+                    <span class="text-primary">El Shadai</span>
+                </h1>
+                <div class="subheading mb-5">
+                    Cruz das Almas - Bahia ·
+                    <a href="mailto:name@email.com"> guinchoelshadailtda@gmail.com </a>
+                </div>
+                <p class="lead mb-5" >  1º paragrafo sobre a empresa </p>
+                <p class="lead mb-5" > 2º paragrafo  sobre a empresa </p>
+            </div>
+        </section>
+        <hr class="m-0" />
+        <!-- Contate-nos-->
+        <section class="resume-section" id="Contate-nosexperience">
+            <div class="resume-section-content">
+                <h2 class="mb-5">Contate-nos</h2>
+                <div class="d-flex flex-column flex-md-row justify-content-between">
+                    <div class="flex-grow-1">
+                        <h3 class="mb-0">Entre em contato pelo :</h3>
+                        <div class="subheading mb-3"> (75) 98113-3180 </div>
+                        
+                    </div>
+                </div>
+            </div>
+        </section>
+        <hr class="m-0" />
+
+    </div>
+    <!-- Bootstrap core JS-->
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js"></script>
+    <!-- Core theme JS-->
+    <script src="js/scripts.js"></script>
```

Comment:
The file path and actual file names are different. Please update this.

Reply:
Done. Thanks for the review!

Reply:
Perfect!